apps/vmq_server/src/vmq_mqtt_fsm_util.erl

```diff
@@ -144,6 +144,8 @@ terminate_reason(?DISCONNECT_MIGRATION) -> normal;
 terminate_reason(?NORMAL_DISCONNECT) -> normal;
 terminate_reason(?SESSION_TAKEN_OVER) -> normal;
 terminate_reason(?REMOTE_SESSION_TAKEN_OVER) -> normal;
+terminate_reason(?INVALID_PUBREC_ERROR) -> normal;
+terminate_reason(?INVALID_PUBCOMP_ERROR) -> normal;
```

Comment:
Map new REASONS to normal. Also where you have changed normal to specific reasons, normal mapping must be done here.

Reply:
Done